app/translations/index.js

```diff
@@ -6,8 +6,14 @@ import Translation from './src/Translation';
 import TranslationFragment from './src/TranslationFragment';
 import DateFormatter from './src/DateFormatter';
 import MapLocaleToLanguageQuery from './src/MapLocaleToLanguageQuery';
+import GetDeviceLocale from './src/GetDeviceLocale';
 
-export { TranslationFragment, DateFormatter, MapLocaleToLanguageQuery };
+export {
+  TranslationFragment,
+  DateFormatter,
+  MapLocaleToLanguageQuery,
+  GetDeviceLocale,
```

Comment:
Should we rename this package to the "localization"? (not in this PR) Name "translations" seems to be too specific.

Reply:
OK: https://github.com/kiwicom/react-native-app/issues/484